File: ejy_service/utils/state_handler.py
class State:
    def success(*args, **kwargs):
        if not args:
            args = ("OK",)
        if not kwargs:
            kwargs["data"] = {}
        return {"state": 1, "msg": args[0], "data": kwargs["data"]}

    def fail(*args):
        if not args:
            args = ["error"]
        return {"state": 0, "msg": args[0]}

File: ejy_pc/WinFormsApp1/utils/StateHandler.cs
using Newtonsoft.Json.Linq;
using System;
using System.Collections.Generic;
using System.Text;

namespace EnjoyPrint.utils
{
    class State
    {
        public bool Success;

        public JObject Data { get; set; }
        public string Msg { get; set; }
    }
}
